porcupine/plugins/python_venv.py

```diff
@@ -21,14 +21,18 @@
 
 
 def is_venv(path: Path) -> bool:
-    landmarks = [path / "pyvenv.cfg"]
-    if sys.platform == "win32":
-        landmarks.append(path / "Scripts" / "python.exe")
-        landmarks.append(path / "Scripts" / "activate.bat")
+    if not "porcupine-venv" in str(path):
```

Comment:
(I tried to include this in my previous preview, but looks like github is doing an April Fools prank on me.)

Let's just check the name of the folder:

```suggestion
    if path.name != "porcupine-venv":
```

The `.name` works like this:

```
>>> import pathlib
>>> pathlib.Path("/home/akuli/porcupine-venv").name
'porcupine-venv'
```

This way, if you make a project named `porcupine-venv-foobar`, you can still use envs inside that project.

Reply:
That is better, thanks!